packages/tasks/src/snippets/js.ts

```diff
@@ -139,9 +115,15 @@ export const jsSnippets: Partial<Record<PipelineType, (model: ModelData, accessT
 };
 
 export function getJsInferenceSnippet(model: ModelData, accessToken: string): string {
-	return model.pipeline_tag && model.pipeline_tag in jsSnippets
-		? jsSnippets[model.pipeline_tag]?.(model, accessToken) ?? ""
-		: "";
+	const body =
+		model.pipeline_tag && model.pipeline_tag in jsSnippets
+			? jsSnippets[model.pipeline_tag]?.(model, accessToken) ?? ""
+			: "";
+
+	return `const API_URL = "https://api-inference.huggingface.co/models/${model.id}"
+const HEADERS = {"Authorization": ${accessToken ? `"Bearer ${accessToken}"` : `f"Bearer {API_TOKEN}"`}}
```

Comment:
If we define `HEADERS` here, shouldn't we reuse it in `fetchSnippet`?
To do that, you would need to define "json headers" and "non-json headers" separately and inject the correct one depending on the task.